### DISCUSSION TITLE
Tileboard Camera Support

### DISCUSSION THREAD

**Discussion by tekwreck** [tsightler/ring-mqtt#594] [Support Q&A]
So I am using Tileboard with this integration to show my ring cameras. The ID is set for snapshot and then the camera is set to the actual camera entity.

It works flawlessly on Tileboard however now in the ring app I am looking at history and its just bombarded with Live Views even though I don't have the feed open on TileBoard.

![image](https://user-images.githubusercontent.com/5428764/227739814-7cfb17c8-5fd8-47a6-a1eb-d6e90767f7e5.png)

Has anyone else seen this and any sort of a solution to it?

Thanks.

**Comment by tsightler** [comment]
I don't know anything at all about tileboard, but it would seem pretty clear that it is opening a stream, or something else is.   Logs should tell you what IP the streaming request is coming from, and the best you can do is try to track it down from there.

**Comment by tekwreck** [comment]
I'll have to go through logs and update from here. I disabled Tileboard and the live stream keeps turning on and off in Home ssistant.
